code/game/turfs/simulated/wall_types.dm

```diff
@@ -255,6 +255,11 @@
 	hard_corner = 1
 	stripe_color = "#00FF00"
 
+/turf/simulated/shuttle/wall/voidcraft/hard_corner/red

```

Comment:
add //YW Addition at beginning of the turf, or put it in a new file called wall_typed_yw.dm